src/components/DoubleInput/DoubleInput.js

```diff
@@ -19,80 +18,68 @@ type Props = {|
 */
 
 type Props = {|
-  +onChange: (value: string) => void,
+  +onChangeLeft: (value: string) => void,
+  +onChangeRight: ?((value: string) => void),
   +onClose: ?(() => void),
-  +valueFiat: string,
-  +valueAmount: string,
-  +errorMessageAmount: string,
+  +valueLeft: string,
+  +valueRight: string,
+  +placeholderLeft: string,
+  +placeholderRight: string,
+  +errorMessage: string,
   +isLoading: boolean,
```

Comment:
Loading can happen for both inputs only?

Reply:
In design we have only one loader, I think in send asset form thereis only one place with this loader (when fiat conversion rate is loading)